src/core/server/ui_settings/integration_tests/index.test.ts

```diff
@@ -23,7 +22,6 @@ describe('uiSettings/routes', function () {
   beforeAll(startServers);
   /* eslint-disable jest/valid-describe */
   describe('doc missing', docMissingSuite(savedObjectIndex));
-  describe('doc missing and index readonly', docMissingAndIndexReadOnlySuite(savedObjectIndex));
   describe('doc exists', docExistsSuite(savedObjectIndex));
```

Comment:
So... I have no idea why, as the PR isn't impacting anything functional, especially not uiSettings related, but I got a lot of flakiness on this suite. 

I spent a few hours trying to understand what was happening, but to be honest I didn't really found what could be causing this. Now, these seems like very old integration tests that were just copied from legacy a long time ago, and I don't think having a integration test suite on the behavior of the uiSettings on a read-only kibana index really make any sense now (I mean, a whole lot of other stuff would explode first, and it's not supported), so I simply removed it.

The other suites don't make much more sense, as what they test is already covered in api-integration test suites now, but as they were passing, I kept them.

Reply:
If everything is covered in api-integration tests, let's remove these tests